### PR TITLE
add _targets file

### PR DESCRIPTION
**Pull request by merrittharlan**
Create and targets makefile

### CODE REVIEW

**Comment by merrittharlan** [comment]
Capture

File: _targets.R [hunk 1]
```diff
@@ -0,0 +1,36 @@
+library(targets)
+source("code.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
```
[the remark this answers is not in the document]
Comment (padilla410) [reply _targets.R:11]: Howdy @merrittharlan, it looks to me like you successfully implemented all of the suggestions from the bot. When coding with others I typically like to leave a comment  for each line suggestion and/or hit "resolve conversation"

**Comment by padilla410** [comment]
Nice job! This ran for me and the outputs look good. Please feel free to merge when you're ready.

image

**Review by padilla410 — APPROVED** [review]

File: _targets.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (merrittharlan) [reply _targets.R:11]: Thanks @padilla410 good to know, I will mark these as resolved

**Pull request merged**